From 8c51792f28c0eb5f0be1c38ac83b6168a9bf0b16 Mon Sep 17 00:00:00 2001
From: Carlos De Maine <carlosd.kde@gmail.com>
Date: Fri, 15 Nov 2024 03:16:27 +0000
Subject: [PATCH] it's Taglib

---
 CMakeLists.txt     | 2 +-
 src/CMakeLists.txt | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c48974d9..0dd279e2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -49,7 +49,7 @@ find_package(MauiKit4 REQUIRED COMPONENTS FileBrowsing)
 if(ANDROID)
     find_package(taglib 1.4 REQUIRED)
 else()
-    find_package(TagLib 1.4 REQUIRED)
+    find_package(Taglib 1.4 REQUIRED)
 endif()
 
 if(UNIX AND NOT ANDROID)
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 72e05dd0..84af7ccc 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -131,7 +131,7 @@ target_compile_definitions(${PROJECT_NAME}
 target_include_directories(${PROJECT_NAME} PRIVATE ${Taglib_INCLUDE_DIRS})
 
 set(Taglib_LIBRARIES
-    TagLib::tag)
+    Taglib::Taglib)
 
 target_link_libraries(${PROJECT_NAME}
     PRIVATE
-- 
GitLab

